process_pub_database/get_data_cpdl.py

```diff
@@ -47,110 +123,48 @@ def main():
 	composers = []
 	composer_dict = {}
 
-	df  = pd.read_json('./cpdl.json')
-
-	list_of_titles = df.keys()
-
-
 
 	for title in list_of_titles:
-		count_total+=1
 
-		try:
-			ret_page = md.page(title)
-			page_text = ret_page.html
+		try: 
+			count_total+=1
 
-			if 'MusicXML' in page_text:
-				source = ret_page.url
+			process, source = check_mxl(md, title, ['MusicXML'])
 
-				fp = urllib.request.urlopen(source)
-				mybytes = fp.read()
+			if process:
 
-				mystr = mybytes.decode("utf8")
-				fp.close()
+				title_str = read_source(source)
 
-				if 'CPDL copyright license' in mystr:
+				if 'CPDL copyright license' in title_str:
 					license = 'https://www0.cpdl.org/wiki/index.php/ChoralWiki:CPDL'
 				else:
 					print('Liscence error')
 					import pdb;pdb.set_trace()
```

Comment:
you shouldn't drop into the debugger here, ideally this will run without input, even in the case of an error.